JenkinsfileCommon.groovy

```diff
@@ -9,11 +9,12 @@ String getAppVersion() {
 
 String getArtifactoryPath() {
     // Choose transient artifactory location for all artifacts except those from release branches
-    ARTIFACTORY_ROOT_FOLDER = 'MAAXA-SNAPSHOT'
+    ARTIFACTORY_ROOT_FOLDER = 'pgog-fss-iam-uaa-mvn-snapshot'
```

Comment:
artifactory links in the commit message pointing to ums repo.

Reply:
Good catch. Amended commit